### PR TITLE
[SCAG] Missing Deities

### PR DESCRIPTION
**Pull request by kgirtxd**

### CODE REVIEW

File: supplements/sword-coast-adventurers-guide/deities.xml [hunk 1]
```diff
@@ -0,0 +1,1510 @@
...
+		</description>
+		<setters>
+			<set name="setting">Forgotten Realms</set>
+			<set name="alignment">LE</set>
+			<set name="domains">Arcana, Death</set>
+			<set name="symbol">Broken arrow</set>
+			<set name="gender">male</set>
+		</setters>
+		<sheet>
+			<description>goddess of winter</description>
```
Comment (NickVendel) [inline supplements/sword-coast-adventurers-guide/deities.xml:867]:
```suggestion
			<description>duergar god of magic and slavery</description>
```

File: supplements/sword-coast-adventurers-guide/deities.xml [hunk 2]
```diff
@@ -0,0 +1,1510 @@
...
+		</sheet>
+	</element>
+	<element name="Marthammor Duin" type="Deity" source="Sword Coast Adventurer’s Guide" id="ID_WOTC_SCAG_DEITY_MARTHAMMOR_DUIN">
+		<description>
+			<p class="flavor">god of wanderers</p>
+		</description>
+		<setters>
+			<set name="setting">Forgotten Realms</set>
+			<set name="alignment">NG</set>
+			<set name="domains">Arcana, Trickery</set>
```
Comment (NickVendel) [inline supplements/sword-coast-adventurers-guide/deities.xml:877]:
```suggestion
			<set name="domains">Nature, Trickery</set>
```

File: supplements/sword-coast-adventurers-guide/deities.xml [hunk 3]
```diff
@@ -0,0 +1,1510 @@
...
+			<set name="symbol">Female drow hand wearing many silver rings</set>
+			<set name="gender">female</set>
+		</setters>
+		<sheet>
+			<description>goddess of necromancy</description>
+		</sheet>
+	</element>
+	<element name="Lolth" type="Deity" source="Sword Coast Adventurer’s Guide" id="ID_WOTC_SCAG_DEITY_LOLTH">
+		<description>
+			<p class="flavor">goddess of spider</p>
```
Comment (NickVendel) [inline supplements/sword-coast-adventurers-guide/deities.xml:1146]:
```suggestion
			<p class="flavor">goddess of spiders</p>
```

File: supplements/sword-coast-adventurers-guide/deities.xml [hunk 4]
```diff
@@ -0,0 +1,1510 @@
...
+			<set name="domains">Life</set>
+			<set name="symbol">Cornucopia on a shield</set>
+			<set name="gender">male female</set>
+		</setters>
+		<sheet>
+			<description>goddess of fertility and protection</description>
+		</sheet>
+	</element>
+
+	<!--Halfling Pantheon-->
```
Comment (NickVendel) [inline supplements/sword-coast-adventurers-guide/deities.xml:1282]:
```suggestion
	<!--Gnomish Pantheon-->
```

File: supplements/sword-coast-adventurers-guide/deities.xml [hunk 5]
```diff
@@ -0,0 +1,1510 @@
...
+		</description>
+		<setters>
+			<set name="setting">Forgotten Realms</set>
+			<set name="alignment">NE</set>
+			<set name="domains">Death</set>
+			<set name="symbol">White hand, palm outward</set>
+			<set name="gender">male</set>
+		</setters>
+		<sheet>
+			<description>god of stealth and darkness</description>
```
Comment (NickVendel) [inline supplements/sword-coast-adventurers-guide/deities.xml:1507]:
```suggestion
			<p class="flavor">god of death and disease</p>
		</description>
		<setters>
			<set name="setting">Forgotten Realms</set>
			<set name="alignment">NE</set>
			<set name="domains">Death</set>
			<set name="symbol">White hand, palm outward</set>
			<set name="gender">male</set>
		</setters>
		<sheet>
			<description>god of death and disease</description>
```

**Review by NickVendel — CHANGES_REQUESTED** [review]

File: supplements/sword-coast-adventurers-guide/deities.xml [hunk 6]
```diff
@@ -0,0 +1,1510 @@
...
+			<set name="alignment">LN</set>
+			<set name="domains">Death</set>
+			<set name="symbol">Upright skeletal arm holding balanced scales</set>
+			<set name="gender">male</set>
+		</setters>
+		<sheet>
+			<description>god of the dead</description>
+		</sheet>
+	</element>
+	<element name="Kossuth" type="Deity" source="Sword Coast Adventurer’s Guide" id="ID_WOTC_SCAG_DEITY_KELEMVOR">
```
Comment (NickVendel) [inline supplements/sword-coast-adventurers-guide/deities.xml:328]:
```suggestion
	<element name="Kossuth" type="Deity" source="Sword Coast Adventurer’s Guide" id="ID_WOTC_SCAG_DEITY_KOSSUTH">
```

**Review by NickVendel — CHANGES_REQUESTED** [review]

File: supplements/sword-coast-adventurers-guide/deities.xml [hunk 7]
```diff
@@ -0,0 +1,1510 @@
...
+	</element>
+	<element name="Shar" type="Deity" source="Sword Coast Adventurer’s Guide" id="ID_WOTC_SCAG_DEITY_SHAR">
+		<description>
+			<p class="flavor">goddess of darkness and loss</p>
+		</description>
+		<setters>
+			<set name="setting">Forgotten Realms</set>
+			<set name="alignment">NE</set>
+			<set name="domains">Death, Trickery</set>
+			<set name="symbol">Black disk encircled with a border</set>
```
Comment (NickVendel) [inline supplements/sword-coast-adventurers-guide/deities.xml:561]:
```suggestion
			<set name="symbol">Black disk encircled with a purple border</set>
```

File: supplements/sword-coast-adventurers-guide/deities.xml [hunk 8]
```diff
@@ -0,0 +1,1510 @@
...
+			<set name="alignment">CN</set>
+			<set name="domains">Trickery</set>
+			<set name="symbol">Two peering elven eyes</set>
+			<set name="gender">male</set>
+		</setters>
+		<sheet>
+			<description>god of outcasts</description>
+		</sheet>
+	</element>
+	<element name="Henali Celanil" type="Deity" source="Sword Coast Adventurer’s Guide" id="ID_WOTC_SCAG_DEITY_HENALI_CELANIL">
```
Comment (NickVendel) [inline supplements/sword-coast-adventurers-guide/deities.xml:1022]:
```suggestion
	<element name="Hanali Celanil" type="Deity" source="Sword Coast Adventurer’s Guide" id="ID_WOTC_SCAG_DEITY_HANALI_CELANIL">
```

File: supplements/sword-coast-adventurers-guide/deities.xml [hunk 9]
```diff
@@ -0,0 +1,1510 @@
...
+	<element name="Vhaeraun" type="Deity" source="Sword Coast Adventurer’s Guide" id="ID_WOTC_SCAG_DEITY_VHAERAUN">
+		<description>
+			<p class="flavor">god of thieves</p>
+		</description>
+		<setters>
+			<set name="setting">Forgotten Realms</set>
+			<set name="alignment">CE</set>
+			<set name="domains">Trickery</set>
+			<set name="symbol">Black mask with blue glass lenses inset over eyes</set>
+			<set name="gender">female</set>
```
Comment (NickVendel) [inline supplements/sword-coast-adventurers-guide/deities.xml:1183]:
```suggestion
			<set name="gender">male</set>
```

**Review by NickVendel — CHANGES_REQUESTED** [review]

**Review by NickVendel — APPROVED** [review]

File: supplements/sword-coast-adventurers-guide/deities.xml [hunk 10]
```diff
@@ -0,0 +1,1510 @@
...
+	</element>
+	<element name="Hoar" type="Deity" source="Sword Coast Adventurer’s Guide" id="ID_WOTC_SCAG_DEITY_HOAR">
+		<description>
+			<p class="flavor">god of revenge and retribution</p>
+		</description>
+		<setters>
+			<set name="setting">Forgotten Realms</set>
+			<set name="alignment">LN</set>
+			<set name="domains">War</set>
+			<set name="symbol">A coin with a two-faced heard</set>
```
Comment (JayPea00101010) [inline supplements/sword-coast-adventurers-guide/deities.xml:261]: Typo, 'heard' -> 'head

File: supplements/sword-coast-adventurers-guide/deities.xml [hunk 11]
```diff
@@ -0,0 +1,1510 @@
...
+			<set name="symbol">Upright skeletal arm holding balanced scales</set>
+			<set name="gender">male</set>
+		</setters>
+		<sheet>
+			<description>god of the dead</description>
+		</sheet>
+	</element>
+	<element name="Kossuth" type="Deity" source="Sword Coast Adventurer’s Guide" id="ID_WOTC_SCAG_DEITY_KOSSUTH">
+		<description>
+			<p class="flavor">god of the fire</p>
```
Comment (JayPea00101010) [inline supplements/sword-coast-adventurers-guide/deities.xml:330]: Typo, 'god of the fire' -> 'god of fire'
Same mistake on line 340

File: supplements/sword-coast-adventurers-guide/deities.xml [hunk 12]
```diff
@@ -0,0 +1,1510 @@
...
+	</element>
+	<element name="Hanali Celanil" type="Deity" source="Sword Coast Adventurer’s Guide" id="ID_WOTC_SCAG_DEITY_HANALI_CELANIL">
+		<description>
+			<p class="flavor">goddess of love and beauty</p>
+		</description>
+		<setters>
+			<set name="setting">Forgotten Realms</set>
+			<set name="alignment">CG</set>
+			<set name="domains">Life</set>
+			<set name="symbol">Golden Heart</set>
```
Comment (JayPea00101010) [inline supplements/sword-coast-adventurers-guide/deities.xml:1030]: typo, 'Heart' -> 'heart' (inkeeping with only first letter uppercase for other symbols)

File: supplements/sword-coast-adventurers-guide/deities.xml [hunk 13]
```diff
@@ -0,0 +1,1510 @@
...
+	<!--Drow Pantheon-->
+	<element name="Eilistraee" type="Deity" source="Sword Coast Adventurer’s Guide" id="ID_WOTC_SCAG_DEITY_EILISTRAEE">
+		<description>
+			<p class="flavor">goddess of song and moonlight</p>
+		</description>
+		<setters>
+			<set name="setting">Forgotten Realms</set>
+			<set name="alignment">CG</set>
+			<set name="domains">Light, Nature</set>
+			<set name="symbol">Sword-weilding dancing drow female silhouetted against the full moon</set>
```
Comment (JayPea00101010) [inline supplements/sword-coast-adventurers-guide/deities.xml:1122]: Typo, 'weilding' -> 'wielding'

File: supplements/sword-coast-adventurers-guide/deities.xml [hunk 14]
```diff
@@ -0,0 +1,1510 @@
...
+	<element name="Yondalla" type="Deity" source="Sword Coast Adventurer’s Guide" id="ID_WOTC_SCAG_DEITY_YONDALLA">
+		<description>
+			<p class="flavor">goddess of fertility and protection</p>
+		</description>
+		<setters>
+			<set name="setting">Forgotten Realms</set>
+			<set name="alignment">LG</set>
+			<set name="domains">Life</set>
+			<set name="symbol">Cornucopia on a shield</set>
+			<set name="gender">male female</set>
```
Comment (JayPea00101010) [inline supplements/sword-coast-adventurers-guide/deities.xml:1275]: Typo, 'male female' -> 'female'

**Review by JayPea00101010 — CHANGES_REQUESTED** [review]

File: supplements/sword-coast-adventurers-guide/deities.xml [hunk 15]
```diff
@@ -0,0 +1,1510 @@
...
+		</description>
+		<setters>
+			<set name="setting">Forgotten Realms</set>
+			<set name="alignment">N</set>
+			<set name="domains">Light</set>
+			<set name="symbol">Flame</set>
+			<set name="gender">male</set>
+		</setters>
+		<sheet>
+			<description>Faerunian god of the fire</description>
```
Comment (JayPea00101010) [inline supplements/sword-coast-adventurers-guide/deities.xml:340]:
```suggestion
			<p class="flavor">god of fire</p>
		</description>
		<setters>
			<set name="setting">Forgotten Realms</set>
			<set name="alignment">N</set>
			<set name="domains">Light</set>
			<set name="symbol">Flame</set>
			<set name="gender">male</set>
		</setters>
		<sheet>
			<description>Faerunian god of fire</description>
```

**Review by JayPea00101010 — APPROVED** [review]

**Pull request merged**